doc/man3/SSL_COMP_add_compression_method.pod

```diff
@@ -94,6 +100,8 @@ L<ssl(7)>
 The SSL_COMP_free_compression_methods() function was deprecated in OpenSSL 1.1.0.
 The SSL_COMP_get0_name() and SSL_comp_get_id() functions were added in OpenSSL 1.1.0d.
 
+All functions described here were deprecaed in OpenSSL 3.0.
+
 =head1 COPYRIGHT
 
 Copyright 2001-2020 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
suggestion: update to 2021...

Reply:
This will get updated automatically before releasing so no problem with that.